src/Compilers/VisualBasic/Portable/Parser/ParserFeature.vb

```diff
@@ -32,24 +32,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         DigitSeparators
         BinaryLiterals
         Tuples
-        IOperation
         InferredTupleNames
         LeadingDigitSeparator
         NonTrailingNamedArguments
         PrivateProtected
     End Enum
 
     Friend Module FeatureExtensions
-        <Extension>
```

Comment:
Please do not remove this from code from the compiler. It is intended to facilitate experimental code residing in branches of the compiler, even though the product code does not contain experiments.